tests/restrictededitingmodeediting.js

```diff
@@ -17,7 +17,7 @@ import LinkEditing from '@ckeditor/ckeditor5-link/src/linkediting';
 import Typing from '@ckeditor/ckeditor5-typing/src/typing';
 import Clipboard from '@ckeditor/ckeditor5-clipboard/src/clipboard';
 
-import RestrictedEditingModeEditing, { onKeyDown } from './../src/restrictededitingmodeediting';
+import RestrictedEditingModeEditing, { onSelectAll } from './../src/restrictededitingmodeediting';
```

Comment:
There's no `onSelectAll` export in that file (which is good).

Side note: I wonder why this doesn't blow up... damn JS :/

Reply:
TypeScript would help us here 👀

Reply:
Would be lovely but https://github.com/ckeditor/ckeditor5/issues/504.